docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+    <calculated-listen-ports>false</calculated-listen-ports>
+    <server-name-prefix>managed-server</server-name-prefix>
+    <dynamic-cluster-size>5</dynamic-cluster-size>
+    <max-dynamic-cluster-size>5</max-dynamic-cluster-size>
+  </dynamic-servers>
+</cluster>
+```
+
+And here is a snippet of off-line WLST that corresponds to the above config.xml snippet:
+
+```
+  templateName = "cluster-1-template"
+  cd('/ServerTemplates/%s' % templateName)
+  templateChannelName = "MyChannel"
+  create(templateChannelName, 'NetworkAccessPoint')
+  cd('NetworkAccessPoints/%s' % templateChannelName)
+  set('Protocol', 't3')
+  set('ListenPort', 7999)
+  set('PublicPort', 30999)
+  set('HttpEnabledForThisProtocol', true)
+  set('TunnelingEnabled', true)
+  set('OutboundEnabled', false)
+  set('Enabled', true)
+  set('TwoWaySslEnabled', false)
+  set('ClientCertificateEnforced', false)
+```
+
+In this example:
+
+- WebLogic binds the custom network channel to port 7999 and the default network channel to 8001.
+
+- The operator will automatically create a Kubernetes service named `DOMAIN_UID-cluster-cluster-1` for both the custom and default channel
+
+- Internal clients running in the same Kubernetes cluster as the channel can access the cluster using `t3://DOMAIN_UID-cluster-cluster-1:8001`.
+
+- External clients would be expected to access the cluster using the custom channel using URLs like `t3://some.public.address.com:30999` or, if leveraging tunneling, `http://some.public.address.com:30999`.
+
+
+#### WebLogic Custom Channel Notes 
+
+- Channel configuration for a configured cluster requires configuring the same network-access-point on each server. The Operator currently doesn't test or support network channels that have a different configuration on each server in the cluster.
+
+- Additional steps are required for external clients beyond configuring the custom channel - see [Approaches](#approaches).
+
+### Setting up a NodePort
+
+#### Getting Started
+
+A Kubernetes NodePort exposes a port on each machine that hosts the Kubernetes cluster where the port is accessible from outside of a Kubernetes cluster. This port redirects network traffic to pods within the Kubernetes cluster. Setting up a Kubernetes NodePort is one of multiple approaches for giving external WebLogic clients access to JMS or EJBs (see [Approaches](#approaches) for alternatives).  
+
+If an EJB or JMS service is running on an Admin Server, then you can skip the rest of this section and use the `spec.adminServer.adminService.channels` domain resource attribute to have the Operator create a Node Port for you. See [Reference - Domain resource]({{<relref "/reference/domain-resource/_index.md">}}). On the other hand, if the EJB or JMS service is running in a WebLogic Cluster or stand-alone WebLogic managed server, and you desire to provide access to the service via a NodePort, then the NodePort must be deployed 'manually' - see the following sample and table.
```

Comment:
Don't say "on the other hand", if you have to say anything, use "Otherwise", or just start a new paragraph

Reply:
I don't think we capitalize the "c" in "WebLogic cluster" - check with @rosemarymarano

Reply:
We don't. WebLogic cluster is correct. And, standalone is one word.

Reply:
I've  made both changes for now.  Checking with Rosemary via slack about the "WebLogic cluster".